docs/en/observability/synthetics-command-reference.asciidoc

```diff
@@ -83,6 +83,13 @@ npx @elastic/synthetics push --auth <api-key> --url <kibana-url> --project <id|n
 
 `--auth`::
 API key used for {kibana-ref}/api-keys.html[{kib} authentication].
++
+If you are pushing to a <<private-locations,private location>>, you must use an API key generated in 8.4 or higher.
++
+To create an API key, you must be logged into {kib} as a user with the following privileges:
++
+* {ref}/security-privileges.html#privileges-list-cluster[Cluster privileges]: `monitor`, `read_ilm`, `read_pipeline`, and at least one of `manage_own_api_key`, `manage_security`, or `manage_api_key`
+* {ref}/security-privileges.html#privileges-list-indices[Indices privileges] for `synthetics-*`: `view_index_metadata`, `create_doc`, `auto_configure`
```

Comment:
@dominiqueclarke I'm not sure if the requirements for creating an API key are identical to those added in #2148. If they aren't can you suggest changes? Thanks!

Reply:
Hi @colleenmcginnis 

Sorry I missed this. The privileges are completely different, unfortunately, and nuanced.

To generate an api key, you need `manage_own_api_key`, `manage_security`, or `manage_api_key` cluster permissions.

```
* {ref}/security-privileges.html#privileges-list-cluster[Cluster privileges]: At least one of `manage_own_api_key`, `manage_security`, or `manage_api_key`
```

Plus AT LEAST Kibana Uptime all permissions if you want to use ONLY PUBLIC LOCATIONS

![image](https://user-images.githubusercontent.com/11356435/189035984-e1a635ed-ac81-45a7-8bec-e02822603b77.png)

If you want to use PRIVATE LOCATIONS, you also need Fleet and Integrations Kibana permissions
![image](https://user-images.githubusercontent.com/11356435/189036146-3278aea9-cad5-4398-9e89-59530c3c0779.png)

Reply:
@dominiqueclarke does this look correct? 

<img width="850" alt="Screen Shot 2022-09-08 at 3 16 58 PM" src="https://user-images.githubusercontent.com/10479155/189217502-3664046e-148c-4842-b9f8-185746c49387.png">